easybuild/tools/job/gc3pie.py

```diff
@@ -69,7 +69,7 @@ class AbortingDependentTaskCollection(AbortOnError, DependentTaskCollection):
         """
         pass
 
-except ImportError as err:
+except ImportError:
```

Comment:
I would restore this, and actually mention the error in the log message:

```python
except ImportError as err:
    _log.debug("Failed to import gc3libs from GC3Pie ('%s'). " % err
               "Silently ignoring, this is a real issue only when GC3Pie is used as backend for --job")

Reply:
Automatic change w/o behavior change. Can be done later